### PR TITLE
Fix backup and recover

### PR DESCRIPTION
**Pull request by rusko124**
Fixed backup and recover for resin.io and Docker.

#800

### CODE REVIEW

File: ansible/roles/screenly/files/screenly_utils.sh [hunk 1]
```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
 
 recover(){
-    sudo service screenly-viewer stop
-    rm -rf /home/pi/.screenly /home/pi/screenly_assets
-    tar -xvf "$1" -C /home/pi/
-    sudo service screenly-viewer start
+    service screenly-viewer stop
```
Comment (vpetersson) [inline ansible/roles/screenly/files/screenly_utils.sh]: Did you try this on a non-resin environment?

Comment (rusko124) [reply ansible/roles/screenly/files/screenly_utils.sh]: Not yet, I will try this soon

Comment (vpetersson) [reply ansible/roles/screenly/files/screenly_utils.sh]: Cool. Let me know as I have a feeling it will break w/out `sudo` on Raspbian.

Comment (vpetersson) [reply ansible/roles/screenly/files/screenly_utils.sh]: That said - i really don't like being able to run as root.

Comment (rusko124) [reply ansible/roles/screenly/files/screenly_utils.sh]: @vpetersson 
Tests were successful.
RaspberryPi 3 B+

**Comment by vpetersson** [comment]
@rusko124 Please reference the issue too so that there is a reference, such as "Closes #X"

**Comment by rusko124** [comment]
@vpetersson Okay, i updated the issue's comment.

**Comment by rusko124** [comment]
I'll test again and give the final answer tomorrow.

**Comment by mrrsocks** [comment]
@vpetersson I checked it again, and I'm sure it works, but restart service manually without sudo has not been successful.

**Comment by rusko124** [comment]
@vpetersson  I checked it again, and I'm sure it works, but restart service manually without sudo has not been successful.

**Pull request closed without merging**